### PR TITLE
checklist created

### PR DESCRIPTION
**Pull request by Y-Havryliv**

### CODE REVIEW

**Review by DmytroSvirsa — CHANGES_REQUESTED** [review]
students always forget to use modules let's write something like this - when you work with a big project it's a good idea to separate it into different modules

**Comment by Y-Havryliv** [comment]
> yeah, sure, I didn't write it, because it's in README already, so I thought to skip it

File: checklist.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+def check_temperature(temperature: int) -> int:
+    if temperature <= 10:
+        temperature = 10
+
+    return temperature
+
+morning_temprature = 9
+evening_temperature = 22
+
+check_temperature(morning_temprature)
```
Comment (Abnormaltype) [inline checklist.md:19]: add the output, please

File: checklist.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+if "ironing_shirts" in home_chores:
+    home_chores["ironing_shirts"] = "done"
+```
+
+3. DRY while creating class instances:
+
+Good example:
+
+```python
+citizens = {"names": ["Dima", "Dania"], "surnames": ["Smith", "Green"]}
```
Comment (Abnormaltype) [inline checklist.md:73]: please, use indentation

File: checklist.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+
+for i in range(len(citizens)):
+    citizens_list.append(Citizen(name=citizens["names"][i], surname=citizens["surnames"][i]))
+```
+
+Bad example:
+
+```python
+citizens = {"names": ["Dima", "Dania"], "surnames": ["Smith", "Green"]}
```
Comment (Abnormaltype) [inline checklist.md:91]: please, use indentation

File: checklist.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+```python
+def get_full_name(first_name: str, last_name: str) -> str:
+    return f"{first_name} {last_name}"
+```
+
+Bad example:
+
+```python
+def get_full_name(x: str, y: str) -> str:
+    return f"{x} {y}"
```
Comment (Abnormaltype) [inline checklist.md:121]:
```suggestion
    return f'{x} {y}'
```

File: checklist.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+```
+
+3. Use annotation, it is a good practice:
+
+Good example:
+
+```python
+def multiply_by_2(number: int) -> Union[int, float]:
```
Comment (Abnormaltype) [inline checklist.md:129]:
```suggestion
def multiply_by_2(number: int) -> int:
```
my bad, I thought, the number would be float

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: checklist.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+Good example:
+
+```python
+knights_preparation = "we are preparing"
+```
+
+Bad example:
+
+```python
+knightsPreparation = "we are preparing"
```
Comment (Abnormaltype) [inline checklist.md:158]: and add -  `KnightsPreparation` also incorrect

**Review by Abnormaltype — APPROVED** [review]
approved with suggestion

File: checklist.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
+# Сheck Your Code Against the Following Points
+
+## Don`t Repeat Yourself
+
+1. Create functions when needed:
```
Comment (alenatovstukha) [inline checklist.md:5]: 1. Create functions when needed.

File: checklist.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+evening_temperature = 22
+
+if morning_temperature <= 10:
+    morning_temperature = 10
+
+if evening_temperature <= 10:
+    evening_temperature = 10
+```
+
+2. Use for loops:
```
Comment (alenatovstukha) [inline checklist.md:36]: 2. Use the `for` loops.

File: checklist.md [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+}
+
+if "cleaning" in home_chores:
+    home_chores["cleaning"] = "done"
+    
+if "ironing_shirts" in home_chores:
+    home_chores["ironing_shirts"] = "done"
+```
+
+3. DRY while creating class instances:
```
Comment (alenatovstukha) [inline checklist.md:68]: DRY while creating class instances.

DRY - это термин такой или вы просто слово с больших букв написали?

File: checklist.md [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+        self.surname = surname
+
+
+citizens_list.append(Citizen(name=citizens["names"][0], surname=citizens["surnames"][0]))
+citizens_list.append(Citizen(name=citizens["names"][1], surname=citizens["surnames"][1]))
+```
+
+## Code Style
+
+1. Make sure you use the double quotes everywhere.
```
Comment (alenatovstukha) [inline checklist.md:114]: тут тоже добавьте хороший и плохой примеры

File: checklist.md [hunk 11]
```diff
@@ -0,0 +1,167 @@
...
+2. Use descriptive and correct variable names:
```
Comment (alenatovstukha) [inline checklist.md:115]: 2. Use descriptive and correct variable names.

File: checklist.md [hunk 12]
```diff
@@ -0,0 +1,167 @@
...
+```
+
+Bad example:
+
+```python
+def get_full_name(x: str, y: str) -> str:
+    return f'{x} {y}'
+```
+
+3. Use annotation, it is a good practice:
```
Comment (alenatovstukha) [inline checklist.md:131]: 3. Use annotation, it is a good practice.

File: checklist.md [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+```
+
+Bad example:
+
+```python
+def multiply_by_2(number):
+    return number * 2
+```
+
+4. Use `snake_case`:
```
Comment (alenatovstukha) [inline checklist.md:147]: 4. Use the `snake_case`.

**Review by alenatovstukha — APPROVED** [review]

**Pull request merged**